<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupTab</class>
 <widget class="QWidget" name="SetupTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="Widget" name="widgetInputs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>684</width>
        <height>462</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3"/>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QPushButton" name="btnCreateList">
       <property name="text">
        <string>新建方案</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnDeleteList">
       <property name="text">
        <string>删除方案</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存方案</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnRead">
       <property name="text">
        <string>导入方案</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header>widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
